test/OpenTelemetry.Instrumentation.AspNetCore.Tests/MetricTests.cs

```diff
@@ -76,6 +76,11 @@ void ProcessExport(Batch<MetricItem> batch)
                 response.EnsureSuccessStatusCode();
             }
 
+            // We need to let End callback execute as it is executed AFTER response was returned.
+            // In unit tests environment there may be a lot of parallel unit tests executed, so
+            // giving some breezing room for the End callback to complete
+            await Task.Delay(TimeSpan.FromSeconds(1));
```

Comment:
Is this a fix or a mitigation?

Reply:
given this was how the existing tests did, and has been stable for a long time, we can consider this as a fix. 
(Even though I hate calling "sleeping for a random time period" a "fix")